cypress/support/cypress-support/src/utils.md

```diff
@@ -277,6 +277,13 @@ request - `cy.destroyGlobalObjects(['lifecycleAppObjectList']);
 ### Examples:
 request - `cy.checkForTags(["TAG1","TAG2"]);
 
+## resolveDeviceVariable
```

Comment:
Can you add something in here so a user reading this would know that the `preprocessed device data` is retrieved from fixtures using the env var: deviceMac?

Reply:
Addressed